src/index.scss

```diff
@@ -9,89 +17,48 @@ $info-border-color: darken($info-color, 20%);
 
 body {
     max-width: min(100%, 900px);
-    margin-left: auto;
-    margin-right: auto;
+    @include spacing.h-margins(auto);
 
-    font-family: Arial, Helvetica, sans-serif;
-    font-size: 18px;
+    font-family: fonts.$text;
+    font-size: $default_font_size;
+    line-height: $default_font_size;
 
     box-shadow: 0 0 20px 5px darken($color: white, $amount: 10%);
 
     > footer {
-        text-align: center;
-    }
-}
-
-footer {
-    min-height: 4em;
-}
-
-ul {
-    padding-left: 3em;
-}
-
-p {
-    padding: 1em;
-}
-
-p + p {
-    padding-top: 0px;
-}
-
-@mixin h-padding($padding) {
-    padding-left: $padding;
-    padding-right: $padding;
-}
-
-@mixin list-spacing($selector, $spacing) {
-    #{$selector} {
-        padding-top: $spacing;
-
-        &:first-of-type {
-            padding-top: 0;
+        display: flex;
+        flex-direction: row;
```

Comment:
Could make mixins for `flew-row()` and `flex-column()` for these.

Reply:
Actually, could make these `centered-row()` or `centered-column()`. Flexbox is just how I'm implementing a row. One could implement the same thing by using CSS Grid (roughly) (at least for a column).